substrate/frame/multisig/src/lib.rs

```diff
@@ -453,7 +454,7 @@ pub mod pallet {
 		/// - I/O: 1 read `O(S)`, one remove.
 		/// - Storage: removes one item.
 		#[pallet::call_index(3)]
-		#[pallet::weight(T::WeightInfo::cancel_as_multi(other_signatories.len() as u32))]
+		#[pallet::weight(T::WeightInfo::cancel_as_multi(other_signatories.len() as u32, 0))]
```

Comment:
@ggwpez check

Reply:
Simplified the benchmarks now. The z param does not matter since it is neither in storage or a call argument.

Reply:
@ggwpez thanks! so need to regenerate bench for this pallet once more?